examples/coreutils/Kraftfile.uniq

```diff
@@ -0,0 +1,9 @@
+spec: v0.6
+
+name: uniq 
+
+rootfs: ./Dockerfile
+
+cmd: ["/usr/bin/uniq"]
```

Comment:
Also pass an argument to it from the local filesystem, e.g. `cmd: ["/usr/bin/uniq", "/file.txt"]`